pages/dkp/kommander/2.2/security/distributed-authnz/index.md

```diff
@@ -17,6 +17,8 @@ The DKP UI admin credentials are stored as a secret. They never leave the bounda
 
 The Dex service issues an [OIDC ID token][oidc_id_token] on successful user authentication. Other platform services use the ID token as an authentication proof. User identity to the Kubernetes API server is provided by the [`kube-oidc-proxy`][kube_oidc_proxy] platform service that reads the identity from an ID token. Web requests to DKP UI access are authenticated by the [traefik forward auth][traefik_forward_auth] platform service.
 
+<p class="message--note"><strong>NOTE: </strong>The [`kube-oidc-proxy`][kube_oidc_proxy] authenticating `kubectl` CLI requests uses the Kubernetes API Server go library to verify OIDC ID tokens. The library has hard-coded `email_verified` field validation. If the field is present in the token it have the value `true`, otherwise the [OIDC ID token][oidc_id_token] will not get verified.</p>
```

Comment:
`[`kube-oidc-proxy`][kube_oidc_proxy]` is this intentional?

Reply:
Yes, the second part is a link that is pointing to the kube-oidc-proxy project.